src/js/utils/apiClient.jsx

```diff
@@ -55,9 +56,12 @@ export const handleSuccess = response => response;
 export const handleError = (error) => {
   switch (error.response.status) {
     case 400: {
-      const errorMessages = _.map(_.get(error, 'response.data.errorMessages', ''), errorMessage => `<div>${errorMessage}</div>`);
+      const errorMessages = _.map(_.get(error, 'response.data.errorMessages', ''), errorMessage => `${errorMessage}, `);
```

Comment:
you are right it could be moved out - I haven't gone into the details here, because anyway we'll have to revisit it probably while having tickets for refactoring existing alerts, but thanks for pointing that out